Source/WebGPU/WGSL/AST/ASTTypeName.h

```diff
@@ -177,6 +177,58 @@ class ParameterizedTypeName : public TypeName {
         return std::nullopt;
     }
 
+    static String baseToString(Base base)
+    {
+        switch (base) {
```

Comment:
Why not just index an array of Strings? To catch mismatch between enum and strings?

Reply:
Exactly. Although I guess we could have an assertion that the size of the array is the same as the number of entries in Base, but I thought this was simpler.